theme.js

```diff
@@ -1,5 +1,9 @@
 const theme = {
   fontFamily: "Merriweather",
+  maxWidth: "1200px",
+  margin: "0 auto",
+  paddingLeft: "16px",
+  paddingRight: "16px",
```

Comment:
I think these four lines were deleted from master in a prior PR, possibly after you started your branch.

Reply:
Yes that styling should just be in `Container` now